tests/unittest/_torch/modules/moe/quantize_utils.py

```
@@ -638,11 +638,21 @@ def check_accuracy(self, output, ref_output):
         # under TTP parallel mode (extra bf16 allreduce from TP splitting).
         # Smaller configs (intermediate_size≤1408, top_k≤6) stay within 3%.
+        #
+        # Element-wise activations (Relu2, Silu) amplify FP4 quantization
+        # errors more than gated activations (SwiGLU). Relu2 (x * relu(x))
+        # squares positive values, approximately doubling the relative error
+        # from FP4 quantization noise. Some kernel tactics exhibit ~10%
+        # mismatch under these conditions.
         top_k = getattr(self.routing_method, "top_k", 1)
         error_accumulation = self.intermediate_size * top_k
         if error_accumulation > 10000:
             # High error accumulation (large intermediate_size × top_k)
```

Ruff (0.15.6)

[warning] Comment contains ambiguous `×` (MULTIPLICATION SIGN). Did you mean `x` (LATIN SMALL LETTER X)?

(RUF003)